x-pack/package.json

```diff
@@ -6,6 +6,7 @@
   "license": "Elastic-License",
   "scripts": {
     "kbn": "node ../scripts/kbn",
+    "kbn:bootstrap": "rm -rf ../built_assets/canvasStorybookDLL",
```

Comment:
Curious why is this necessary?

Reply:
We want to force a rebuild of the DLL only if the packages change.  We don't want to rebuild the DLL on every startup, just when the dependencies are updated.